### PR TITLE
Fix OOM exception when there is an infinite loop in an exception cause

### PR DESCRIPTION
**Pull request by rmacor**

### CODE REVIEW

**Comment by rvansa** [comment]
Could you report the suspended exceptions there, too?

**Comment by rmacor** [comment]
PR updated.

File: core/src/main/java/org/radargun/DistStageAck.java [hunk 1]
```diff
@@ -54,9 +58,16 @@ public void setRemoteException(Throwable remoteException) {
             for (StackTraceElement ste : stackTraceElements) {
                sb.append("\n\t at ").append(ste.toString());
             }
+            sb.append("\nsuppressed: ");
+            sb.append(remoteException.getSuppressed().toString());
```
Comment (mcimbora) [inline core/src/main/java/org/radargun/DistStageAck.java]: I think you'll need to use either Arrays.toString or iterate over the elements and print individual StackTraceElements if you want more detailed message (I'd opt for option no. 2).

**Comment by rmacor** [comment]
PR updated.

**Comment by mcimbora** [comment]
@rmacor Thanks Roman, integrated to 2.x branch as well.

**Pull request closed without merging**